package-lock.json

```diff
@@ -0,0 +1,3122 @@
+{
+  "name": "angular-lock",
```

Comment:
all repos use yarn, so let's remove this file. if you want to add a lock file, use yarn.

Reply:
Done